### DISCUSSION TITLE
How to hide github actions/dependabot from homepage

### DISCUSSION THREAD

**Discussion by PythonCoderAS** [community/community#44352] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

My homepage has a lot of commits by GitHub Actions/Dependabot but I don't really care about them and would like to view activity by actual humans. How do I hide activity from github actions and/or dependabot?

**Comment by mmcloughlin** [comment]
I think I have essentially the same question, though for a different bot account.

The contributors page clearly says "Contributions to master, **excluding merge commits and bot accounts**". However my bot user shows up with 45 commits.

https://github.com/mmcloughlin/avo/graphs/contributors

Github appears to know this account is a bot. For example, here's one of the auto-landed PRs where the GUI shows a bot badge next to the app account:

https://github.com/mmcloughlin/avo/pull/353

![Screenshot from 2023-01-14 13-52-11](https://user-images.githubusercontent.com/7133685/212498678-ad0fdcf6-ba04-4d96-9841-f1032d571cd1.png)
